API/resources/__init__.py

```diff
@@ -42,13 +42,10 @@ def config_modules(env):
 
     for module in modules.values():
         for config in module.get('config', []):
-            # Do not configure if the contents of deps
-            # are not in the modules list.
-            deps = config.get('deps', [])
+            # Do not configure if the result of cond is false.
+            cond = config.get('cond', 'True')
```

Comment:
Ah, 'cond' means 'condition', it takes some time to understand it. :)
I think it would be better a more descriptive name for the variable and for the json key as well, e.g.: 'criteria' or something else.